imagetest/test_suites/metadata/setup.go

```diff
@@ -12,40 +12,189 @@ var Name = "metadata"
 
 const (
 	daemonScriptTemplate = `#!/bin/bash
-
-nohup sleep 3600 > /dev/null 2>&1 < /dev/null &
-echo $! > %s
-`
+	nohup sleep 3600 > /dev/null 2>&1 < /dev/null &
+	echo $! > %s
```

Comment:
this script is only used once, why template it?

Reply:
The script is only used once, but the output path value is used more than once.